modules/react/select/lib/SelectCard.tsx

```diff
@@ -1,20 +1,22 @@
 import {Combobox} from '@workday/canvas-kit-react/combobox';
-import {createStyles, px2rem} from '@workday/canvas-kit-styling';
-import {mergeStyles} from '@workday/canvas-kit-react/layout';
+import {createStencil, handleCsProp, px2rem} from '@workday/canvas-kit-styling';
 import {useSelectCard} from './hooks/useSelectCard';
 import {useSelectModel} from './hooks/useSelectModel';
 import {createSubcomponent, ExtractProps} from '@workday/canvas-kit-react/common';
+import {menuCardStencil} from '@workday/canvas-kit-react/menu';
 
-const selectCardStyles = createStyles({
-  maxHeight: px2rem(300),
+export const selectCardStencil = createStencil({
+  base: {
+    [menuCardStencil.vars.maxHeight]: px2rem(300),
+  },
 });
```

Comment:
Do you need any stencil here at all?

Reply:
So I make this a stencil because if someone is using our css kit, they have this class name. If I move it to the render function, that abstraction is lost. Just being consistent with how we style things.

Reply:
make sense